### DISCUSSION TITLE
Automatically trigger webhook based on code directory?

### DISCUSSION THREAD

**Discussion by DreamPWJ** [vercel/community#416] [Idea]
In many cases, the deployment code is part of the total code. I only hope that the build will be triggered automatically after the code directory that needs to be deployed changes, otherwise there will be a lot of invalid builds

**Comment by Pieparker** [comment]
I think you may want to [tackle this from the opposite angle](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel): only build when certain directories change!
